azext_iot/central/commands_enrollment_group.py

```diff
@@ -0,0 +1,300 @@
+
+
+def create_enrollment_group(
+    cmd,
+    app_id: str,
+    attestation: str,
+    display_name: str,
+    type: str,
+    group_id: str,
+    enabled: bool = False,
+    entry: str = None,
+    certificate: str = None,
+    verified: bool = None,
+    etag : str = None,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=API_VERSION,
+) -> EnrollmentGroupGa:
+    provider = CentralEnrollmentGroupProvider(
+        cmd=cmd, app_id=app_id, token=token, api_version=api_version
+    )
+
+    attestation = utility.process_json_arg(attestation, argument_name="attestation")
+
+    response = provider.create_enrollment_group(
+        group_id=group_id,
+        attestation=attestation,
+        display_name=display_name,
+        type=type,
+        enabled=enabled,
+        etag=etag,
+        central_dns_suffix=central_dns_suffix,
+    )
+
+    if None not in (certificate, verified, entry):
+        # We need to set up the primary or secondary x509 certificate
+        response["x509"] = create_x509(
+            cmd=cmd,
+            app_id=app_id,
+            group_id=group_id,
+            entry=entry,
+            certificate=certificate,
+            verified=verified,
+            etag=etag,
+            token=token,
+            central_dns_suffix=central_dns_suffix,
+            api_version=api_version,
+        )
+
+    return response
+
+
+def update_enrollment_group(
+    cmd,
+    app_id: str,
+    group_id: str,
+    attestation: str = None,
+    display_name: str = None,
+    type: str = None,
+    enabled: bool = False,
+    entry: str = None,
+    certificate: str = None,
+    verified: bool = None,
+    remove_x509: bool = None,
+    etag : str = None,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=API_VERSION,
+):
+    provider = CentralEnrollmentGroupProvider(
+        cmd=cmd, app_id=app_id, token=token, api_version=api_version
+    )
+
+    if attestation is not None:
+        attestation = utility.process_json_arg(attestation, argument_name="attestation")
+
+    response = provider.update_enrollment_group(
+        group_id=group_id,
+        attestation=attestation,
+        display_name=display_name,
+        type=type,
+        enabled=enabled,
+        etag=etag,
+        central_dns_suffix=central_dns_suffix,
+    )
+
+    if None not in (certificate, verified, entry):
+        # We need to set up the primary or secondary x509 certificate
+        response["x509"] = create_x509(
+            cmd=cmd,
+            app_id=app_id,
+            group_id=group_id,
+            entry=entry,
+            certificate=certificate,
+            verified=verified,
+            etag=etag,
+            token=token,
+            central_dns_suffix=central_dns_suffix,
+            api_version=api_version,
+        )
+    elif remove_x509 is True:
+        # We need to remove x509 from the group
+        response["x509"] = {
```

Comment:
how do we know which cert to remove?

Reply:
'entry' parameter is either 'primary' or 'secondary'. We can only remove the primary or secondary x509 certificate of an enrollment group.

Reply:
That's way we need certificate-entry parameter to specify 'primary' or 'secondary' from user input.